Automattic/wp-calypso
# Add registration agreement URL link to purchases page.

#### Changes proposed in this Pull Request

* Add a link to the domain registration agreement for domain purchases on the /me/purchases page.
* Show the registration agreement links for each domain in the cart on the checkout page.

#### Testing instructions

Depends on D26087-code

Apply the patch to the back end and visit the /me/purchases page for each of your domains. Make sure that the "Domain Registration Agreement" is visible and is correct for each of your domains.

<img width="738" alt="Screen Shot 2019-03-27 at 10 23 05 AM" src="https://user-images.githubusercontent.com/1379730/55109132-77bcc380-50ab-11e9-9f15-f16a2d670364.png">

Put several domains in the cart (try something like a .com, a .live, and a .co.in) and make sure the correct registration agreement is shown on the checkout page for each domain registration in the cart.

Example: 

<img width="741" alt="Screen Shot 2019-03-28 at 9 16 26 AM" src="https://user-images.githubusercontent.com/1379730/55166390-66c09080-5145-11e9-9c13-6c1b3b693786.png">

Make sure this works on each of the checkout pages.

There is a link to the agreement on the edit contact info page which should also be set to the url for the applicable agreement.

<img width="741" alt="Screen Shot 2019-03-28 at 3 04 06 PM" src="https://user-images.githubusercontent.com/1379730/55185934-cdf13b80-516b-11e9-9f81-a96b8ba8e2f7.png">

And the same language is shown on the Transfer Domain To Another User page:

<img width="741" alt="Screen Shot 2019-03-28 at 3 08 59 PM" src="https://user-images.githubusercontent.com/1379730/55186057-02fd8e00-516c-11e9-9ee6-a0326f65ac80.png">
